### ISSUE TITLE
Does not work on Alpine

### ISSUE THREAD

**Issue by dmikushin** [SomajitDey/tunnel#10]
Hi @SomajitDey , thanks for this work, it's definitely as thoughtful as the work of physicist should be!

I tried my luck to launch tunnel in Alpine, a usual distro for containers:

```
/ # ./tunnel -i
/usr/bin/env: unrecognized option: S
BusyBox v1.36.1 (2023-07-27 17:12:24 UTC) multi-call binary.

Usage: env [-i0] [-u NAME]... [-] [NAME=VALUE]... [PROG ARGS]

Print current environment or run PROG after setting up environment

        -, -i   Start with empty environment
        -0      NUL terminated output
        -u NAME Remove variable from environment
/ # vi tunnel 
/ # ./tunnel -i
env: can't execute 'bash -O extglob': No such file or directory
/ # vi tunnel 
/ # ./tunnel -i
./tunnel: line 344: syntax error near unexpected token `('
./tunnel: line 344: `    +([[:digit:]]) | ipfs )'
```

**Comment by SomajitDey** [comment]
Thanks for your comment. Kindly refer to @ms2048's [comment](https://github.com/SomajitDey/tunnel/issues/6#issue-1461147806) for a fix. Note, however, that, although tunnel would work as expected following this fix, it might throw some non-fatal [errors](https://github.com/SomajitDey/tunnel/issues/6#issuecomment-1494129382) which you must choose to ignore for now.

**Comment by SomajitDey** [comment]
Dear @dmikushin,

Fixed the issue in [v0.2.1](https://github.com/SomajitDey/tunnel/releases/tag/v0.2.1).
